intake_esm/merge_util.py

```diff
@@ -0,0 +1,150 @@
+import fsspec
+import xarray as xr
+
+
+def join_new(dsets, dim_name, coord_value, options={}):
+    concat_dim = xr.DataArray(coord_value, dims=(dim_name), name=dim_name)
+    return xr.concat(dsets, dim=concat_dim, **options)
+
+
+def join_existing(dsets, options={}):
+    return xr.concat(dsets, dim='time')
```

Comment:
Are we keeping this hardwired to 'time' intentionally here?

Reply:
In the esm-collection-spec, we had `options = {'dim_name': 'time'}`. Usage of `options` in this function (if we keep this approach) is distinct from the other aggregators: there must be an entry `'dim_name'`.

Reply:
If we follow my ruthless attempt to standardize the specification, there would be no hard-coded assumptions like this about how to mere the data. All such operations should be specified in the collection.json file, not in the code itself.

Reply:
Agreed. Should be
```python
xr.concat(dsets, dim=options['dim_name'])
```
if we stick with how we prototyped it.

Reply:
> Are we keeping this hardwired to 'time' intentionally here?

No. I forgot to change this line. I will fix it